middlewares/validators/userStatus.js

```diff
@@ -97,10 +97,15 @@ const validateMassUpdate = async (req, res, next) => {
   const schema = Joi.object()
     .keys({
       users: Joi.array()
-        .items(Joi.string().trim())
+        .items(
+          Joi.object({
+            userId: Joi.string().trim().required(),
+            expectedState: Joi.string().valid("IDLE", "ACTIVE").required(),
```

Comment:
Instead of hardcoding `IDLE` and`ACTIVE`, use the constants present in `constants/userStatus.js`